components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -462,14 +475,14 @@ void RewardsServiceImpl::StartLedger() {
   bat_ledger_service_.set_connection_error_handler(
       base::Bind(&RewardsServiceImpl::ConnectionClosed, AsWeakPtr()));
 
-  bool isProduction = true;
+  bool is_production = !ShouldUseStagingServerForAndroid();
```

Comment:
Should be platform agnostic

Reply:
`ShouldUseStagingServerForAndroid` excluded with pre-processor for Android only